modules/tensor_mechanics/doc/content/source/auxkernels/RankTwoScalarAux.md

```diff
@@ -4,376 +4,10 @@
-         block=AuxVariables/sint
-
-## Triaxiality Stress
-
-The scalar type `TriaxialityStress` finds the ratio of the hydrostatic measure, $T_{hydrostatic}$, to
-the von Mises measure, $T_{vonMises}$, as shown in [eq:triaxiality_scalar_type].  As the
-name suggests, this scalar measure is most often used for stress tensors.
-\begin{equation}
-\label{eq:triaxiality_scalar_type}
-s = \frac{T_{hydrostatic}}{T_{vonMises}} = \frac{\frac{1}{3} T_{ii}}{\sqrt{\frac{3}{2} S_{ij} S_{ij}}}
-\end{equation}
-where $S_{ij}$ is the deviatoric tensor of the Rank-2 tensor $T_{ij}$.
-
-## Volumetric Strain
-
-The scalar type `VolumetricStrain` computes the volumetric strain, defined as
-\begin{equation}
-\label{eq:volumetric_strain}
-\varepsilon_{vol} = \frac{\Delta V}{V}
-\end{equation}
-where $\Delta V$ is the change in volume and $V$ is the original volume.
-
-This calculation assumes that the strains supplied as input ($T$) are logarithmic strains,
-which are by definition $log(L/L_0)$, where $L$ is the current length and $L_0$
-is the original length of a line segment in a given direction.
-The ratio of the volume change of a strained cube to the original volume is thus:
-\begin{equation}
-\label{eq:volumetric_strain_from_tensor}
-s = \frac{\Delta V}{V} = \exp(T_{11}) * \exp(T_{22}) * \exp(T_{33}) - 1
-\end{equation}
-This is the value computed as the volumetric strain.
-
-
-!alert! note title=Finite strain effects
-This calculation assumes that the supplied Rank-2 tensor $T_{ij}$ is a logarithmic strain, which is
-the strain quantity computed for finite strain calculations. The small-strain equivalent of this
-calculation would be
-\begin{equation}
-\label{eq:volumetric_strain_small_strain}
-s = T_{11} + T_{22} + T_{33}
-\end{equation}
-which assumes that engineering strains are supplied and ignores higher-order terms. There is currently
-no option to compute this small-strain form of the volumetric strain because at small strains, the
-differences between the finite strain form used and the small strain approximation is small.
-!alert-end!
-
-### Example Input File Syntax
-
-!listing modules/combined/test/tests/internal_volume/rz_displaced.i
-         block=AuxKernels/fred
-
-An AuxVariable is required to store the AuxKernel information. Note that the name of the AuxVariable
-is used as the argument for the `variable` input parameter in the `RankTwoScalarAux` block.
-
-!listing modules/combined/test/tests/internal_volume/rz_displaced.i
-         block=AuxVariables/volumetric_strain
-
-## Von Mises Stress
-
-The scalar type `VonMisesStress` calculates the vonMises measure for a Rank-2 tensor, as shown in
-[eq:vonmises_scalar_type].  This quantity is usually applied to the stress tensor.
-\begin{equation}
-\label{eq:vonmises_scalar_type}
-s = \sqrt{\frac{3}{2} S_{ij} S_{ij}}
-\end{equation}
-where $S_{ij}$ is the deviatoric tensor of the Rank-2 tensor $T_{ij}$.
+This is a namespace that provides a set of functions to compute scalar
```

Comment:
No -- this is the AuxKernel that allows those quantities to be computed using RankTwoScalarTools. This isn't a namespace. I wouldn't use the word `namespace` anywhere in the documentation.  That will just confuse people.

It's fine to just have a short summary like this here and link to RankTwoScalar tools, but those issues need to be straightened out.